pypeit/scripts/run_pypeit.py

```diff
@@ -64,6 +64,9 @@ def get_parser(cls, width=None):
                             action='store_false',
                             help='Do not load previously generated calibrations, even ones made '
                                  'during the run.')
+        parser.add_argument('--no_wave', default=False,
+                            action='store_true',
+                            help='Stop calibration steps before doing wavelength calibration or tilts.')
```

Comment:
Part of the idea here is to allow users to experiment with the slit extraction/edge tracing parameters before commiting to the full calibration steps. This avoids issues due to the wrong set of traces being used, which can cause weird, hard to document errors in the wavelength calibration. But I agree it isn't very generalizable to other steps. We could take it out if y'all don't feel it's necessary.

Reply:
That was the intention but I think it actually does not do that. I think we can probably actually remove this, since the reference=pixel parameter exists